tools/docker/keycloak/.env

```
@@ -0,0 +1,66 @@
+# Configuration of the Keycloak test IdP (OIDC + SAML) and of its wiring into VitamUI.
+# Everything below is read both by docker compose and by ./setup.sh.
+
+# ---------------------------------------------------------------------------
+# Keycloak
+# ---------------------------------------------------------------------------
+KEYCLOAK_VERSION=26.1
+KEYCLOAK_HTTP_PORT=8042
+# URL through which BOTH the browser and the CAS server reach Keycloak.
+# It is embedded in the OIDC discovery document and in the SAML metadata, so it
+# must be identical on both sides.
+KEYCLOAK_BASE_URL=http://localhost:8042
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_BASE_URL key should go before the KEYCLOAK_HTTP_PORT key

(UnorderedKey)